lib/puppet/module.rb

```diff
@@ -353,8 +364,27 @@ def ==(other)
     self.environment == other.environment
   end
 
+  def initialize_i18n
+    locales_path = File.absolute_path('locales', path)
+    config_path = File.absolute_path('config.yaml', locales_path)
+    module_name = @forge_name.gsub("/","-") if @forge_name
+
+    return nil if i18n_initialized? module_name
```

Comment:
I'm also not sure how the method is designed to fail here?  This method rescues ... and so does its caller. Can probably streamline the error-handling a bit.